### PR TITLE
Item tile improvements

### PR DESCRIPTION
**Pull request by dualcnhq**
## Changes:

- Add mediatype-icon height and width style
- Add default mediatype-icon height and width style 
- Add mediatype-icon fillColor
- Initial changes for item-tile styling

### CODE REVIEW

File: src/tiles/grid/item-tile.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export class ItemTile extends LitElement {
               <mediatype-icon
                 .mediatype=${this.model?.mediatype}
                 ?showText=${true}
+                style="--iconHeight: 10px; --iconWidth: 10px;"
```
Comment (jbuckner) [inline src/tiles/grid/item-tile.ts:74]: Can we put this in the CSS `style` section instead of inline?

File: src/mediatype-icon.ts [hunk 2]
```diff
@@ -41,21 +46,44 @@ export class MediatypeIcon extends LitElement {
...
     movies: 'Movie',
+    radio: 'Radio',
     software: 'Software',
     texts: 'Text',
     tv: 'TV',
     video: 'Video',
     web: 'Web',
   };
 
+  private getDisplayMediaType = () => {
```
Comment (jbuckner) [inline src/mediatype-icon.ts:60]: For things that don't require parameters, I would suggest using the `getter` form:
```
private get displayMediaType() {
  ...
  return ...
}

...

const displayType = this.displayMediaType
```

File: src/tiles/grid/item-tile.ts [hunk 3]
```diff
@@ -331,7 +340,8 @@ export class ItemTile extends LitElement {
         color: #333;
         text-decoration: none;
         display: block;
-      }
+      } 
+      */
```
Comment (jbuckner) [inline src/tiles/grid/item-tile.ts:344]: Do you still need this commented style?

File: src/mediatype-icon.ts [hunk 4]
```diff
@@ -3,31 +3,36 @@ import { customElement, property } from 'lit/decorators.js';
...
 import { tvIcon } from './assets/img/icons/mediatype/tv';
 import { videoIcon } from './assets/img/icons/mediatype/video';
 import { webIcon } from './assets/img/icons/mediatype/web';
-import { collectionIcon } from './assets/img/icons/mediatype/collection';
 
 @customElement('mediatype-icon')
 export class MediatypeIcon extends LitElement {
   @property({ type: String }) mediatype: string | undefined;
 
+  @property({ type: Array }) collection: string[] | undefined;
```
Comment (jbuckner) [inline src/mediatype-icon.ts:22]: I might pluralize this since it's an array

File: src/mediatype-icon.ts [hunk 5]
```diff
@@ -44,21 +49,44 @@ export class MediatypeIcon extends LitElement {
...
+    }
+    return this.mediatype;
+  };
+
   render() {
     if (!this.mediatype) {
       return html``;
     }
+
+    const displayMediatype = this.getDisplayMediaType() || '';
```
Comment (jbuckner) [inline src/mediatype-icon.ts:84]: Does `getDisplayMediaType()` return an optional?

File: src/mediatype-icon.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (dualcnhq) [reply src/mediatype-icon.ts:60]: as of now, i put it here since it only affects the mediatype icon display

File: src/mediatype-icon.ts (hunk 5 above)
Comment (dualcnhq) [reply src/mediatype-icon.ts:84]: i'm getting:
```
const displayMediatype: string | undefined
Type 'undefined' cannot be used as an index type.ts(2538)
```

if i don't add `|| ''` in the end

File: src/tiles/grid/item-tile.ts [hunk 6]
```diff
@@ -76,12 +73,17 @@ export class ItemTile extends LitElement {
             <span>${itemCreator}</span>
           </div>
         </div>
+
+        <div class="hr"></div>
```
Comment (jbuckner) [inline src/tiles/grid/item-tile.ts:77]: I think my other comment got lost in the rebase, but do you need a separate div for this horizontal line or can you set a `border-top` on the `#item-stats-container`?

**Review by jbuckner — APPROVED** [review]
Looks good 👍

**Pull request merged**